packages/@webex/contact-center/src/types.ts

```diff
@@ -0,0 +1,108 @@
+export enum HTTP_METHODS {
```

Comment:
There is a new way of creating enums.. can we check the plugin-meetings for this?

Reply:
https://confluence-eng-gpk2.cisco.com/conf/display/WTWC/How+to+use+enums+and+alternatives

Reply:
OK will check

Reply:
Imported enums from calling so this change is needed in calling package